packages/devtools_app/test/service_manager_test.dart

```diff
@@ -72,16 +74,17 @@ void main() async {
             'getFlagList',
             'getVM',
             'getIsolate',
-            'streamListen',
-            'streamListen',
-            'streamListen',
-            'streamListen',
-            'streamListen',
-            'streamListen',
-            'streamListen',
-            'streamListen',
           ]),
         );
+
+        expect(
+          serviceManager.service.vmServiceCalls
+              .where((call) => call == 'streamListen')
+              .toList()
+              .length,
+          equals(2 + 8), // 2 from package:devtools_testing:flutter_test_driver.
```

Comment:
instead of working around the calls from test driver in our test, let's clear out vmServiceCalls once the test driver has finished setup. That will let us remove the test driver calls in the above expect statement too